pkg/api/2019-10-27-preview/convertertointernal.go

```diff
@@ -90,6 +90,9 @@ func mergeProperties(oc *OpenShiftManagedCluster, cs *api.OpenShiftManagedCluste
 	if oc.Properties.FQDN != nil {
 		cs.Properties.FQDN = *oc.Properties.FQDN
 	}
+	if oc.Properties.MasterPoolProfile != nil && oc.Properties.MasterPoolProfile.PrivateAPIServer != nil {
+		cs.Properties.PrivateAPIServer = *oc.Properties.MasterPoolProfile.PrivateAPIServer
+	}
```

Comment:
Is this something that should be updateable? What would the user expect to happen if this was switched from "true" to "false" or vice versa?